packages/react-component-library/cypress/e2e/TabSet/scrollable.cy.ts

```diff
@@ -4,6 +4,7 @@ import selectors from '../../selectors'
 
 const INITIAL_VIEWPORT_WIDTH = 1400
 const VIEWPORT_HEIGHT = 200
+const CLOSE_TO_DELTA = 2
```

Comment:
1 was just for rounding, but for some reason when TabSet is using smooth scrolling it's another pixel off in Firefox, so had to bump this to 2.